src/docs/file-formats.dox

```diff
@@ -117,8 +108,7 @@ considered.   (See @ref compression_considerations for details.)
   Block compression is disabled by default.
 
 Variable-length column stores support four types of compression:
```

Comment:
Change "four" to "three". Alternately change it to "several" to keep it correct if we change the compression types again.

Reply:
Good catch - thanks! Fixed in 714ae27.